rules/do-you-know-how-to-delete-orphaned-users-from-the-people-picker/rule.md

```diff
@@ -11,20 +11,19 @@ redirects: []
 
 ---
 
-The SharePoint "People Picker" is a great tool for quickly selecting users of your SharePoint site. It allows you to quickly select users from your organization by browsing Active Directory. 
+The SharePoint "People Picker" is a great tool for quickly selecting users of your SharePoint site. It allows you to quickly select users from your organization by browsing Active Directory.
 <!--endintro-->
-![](PeoplePicker.jpg)
-**Figure: The People Picker** 
+![Figure: The People Picker](PeoplePicker.jpg)
+**Figure: The People Picker**
 Unfortunately, if you have added users directly to your site collection, and later deleted or disabled them from Active Directory, you will notice that these orphaned users will still appear in your People Picker. This will eventually clutter up your People Picker.
-![](PeoplePicker-deleted.jpg)
-**Figure: A user in the People Picker that was deleted from Active Directory** ![](PeoplePicker-searching.jpg)
-**Figure: Searching in Active Directory for the user shows the user is in fact deleted** 
-To remove these orphaned users, as a SharePoint Administrator, you can open the following URL (where www.northwind.com is your SharePoint URL):
-
-* http://www.northwind.com/\_catalogs/users/simple.aspx
+![Figure: A user in the People Picker that was deleted from Active Directory](PeoplePicker-deleted.jpg)
+**Figure: A user in the People Picker that was deleted from Active Directory** ![Figure: Searching in Active Directory for the user shows the user is in fact deleted](PeoplePicker-searching.jpg)
```

Comment:
@tombui99 

- [ ] 1. Please break a line between these 2 images

Reply:
Hey @tiagov8 

Done - Added line break between the 2 images